lib/libssh2

```diff
@@ -1 +1 @@
-Subproject commit 6d553a7bb94966466c4db0abf9a26660e786a275
+Subproject commit 88366b5ec26c1ac16981e81201cabfb3a6bff8b1
```

Comment:
I think you can resolve this by doing `git checkout  libssh2-1.7.0` from lib/libssh2.